### PR TITLE
add initial reference for CGA (chainguard)

### PR DESCRIPTION
**Pull request by cpanato**
cc @mattmoor @luhring

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -387,6 +387,17 @@ The defined database prefixes and their "home" databases are:
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>CGA</code></td>
+      <td><a href="https://github.com/wolfi-dev/advisories">CGA / Wolfi - Chainguard Security Notices</a></td>
+      <td>
+        <ul>
+          <li>How to contribute: TBD</li>
```
Comment (mattmoor) [inline docs/schema.md:395]: I’d link to a CONTRIBUTING.md in the advisories repo, and then we can flesh that out

File: README.md [hunk 2]
```diff
@@ -47,6 +48,7 @@ Together, these include vulnerabilities from:
 -   Rocky Linux
 -   RubyGems
 -   Ubuntu
+-   Chainguard
```
Comment (oliverchang) [inline README.md:51]: Would you mind keeping this list alphabetically sorted?

File: README.md [hunk 3]
```diff
@@ -20,6 +20,7 @@ This is the repository for the Open Source Vulnerability schema (OSV Schema), wh
 - [Rust Advisory Database](https://github.com/RustSec/advisory-db)
 - [Ubuntu](https://github.com/canonical/ubuntu-security-notices/)
 - [VMWare Photon OS](https://github.com/vmware/photon/wiki/Security-Advisories) (unofficial)
+- [Chainguard](https://packages.cgr.dev/chainguard/osv/all.json)
```
Comment (oliverchang) [inline README.md:23]: Would you mind keeping this list alphabetically sorted?

**Review by oliverchang — COMMENTED** [review]
Thanks @cpanato !!!! Just some comments.

File: docs/schema.md [hunk 4]
```diff
@@ -636,6 +647,7 @@ The defined ecosystems are:
 | `Bioconductor` | The biological R package ecosystem. The `name` is an R package name. |
 | `Bitnami` | Bitnami package ecosystem; the `name` is the name of the affected component. |
 | `ConanCenter` | The ConanCenter ecosystem for C and C++; the `name` field is a Conan package name.  |
+| `CGA` | Chainguard package ecosystem; the `name` is the name of the affected component. |
```
[the remark this answers is not in the document]
Comment (cpanato) [reply docs/schema.md:650]: The dataset I've sent to start ingesting is a join of Wolfi + Chainguard, but the ecosystem is `chainguard` for all of them. 
Only `wolfi` we will work soon, and then we will add that here as well

thanks for the feedback

[the remark this answers is not in the document]
Comment (cpanato) [reply docs/schema.md]: applied some changes

File: docs/schema.md [hunk 5]
```diff
@@ -636,6 +647,7 @@ The defined ecosystems are:
 | `Bioconductor` | The biological R package ecosystem. The `name` is an R package name. |
 | `Bitnami` | Bitnami package ecosystem; the `name` is the name of the affected component. |
 | `ConanCenter` | The ConanCenter ecosystem for C and C++; the `name` field is a Conan package name.  |
+| `CGA` | Chainguard package ecosystem; the `name` is the name of the package name. The package name might have a `:<RELEASE>` suffix to scope the package to a particular version stream. `<RELEASE>` is a numeric version. |
```
Comment (oliverchang) [inline docs/schema.md:650]: This string needs to match exactly what comes in the `package` entry. i.e. "Chainguard" here per

```
"package": {
        "ecosystem": "Chainguard",
        "name": "eks-distro-coredns-1.8",
        "purl": "pkg:apk/chainguard/eks-distro-coredns-1.8"
      },
```

Also, does Chainguard/Wolfi have a concept of source vs binary package names? If so, we should call out which this is here.

Comment (cpanato) [reply docs/schema.md:650]: i will confirm, but i think all is a binary package

File: docs/schema.md [hunk 6]
```diff
@@ -636,6 +647,7 @@ The defined ecosystems are:
 | `Bioconductor` | The biological R package ecosystem. The `name` is an R package name. |
 | `Bitnami` | Bitnami package ecosystem; the `name` is the name of the affected component. |
 | `ConanCenter` | The ConanCenter ecosystem for C and C++; the `name` field is a Conan package name.  |
+| `CGA` | The Chainguard package ecosystem; the `name` is the name of the source package. The package name might have a `:<RELEASE>` suffix to scope the package to a particular version stream. `<RELEASE>` is a numeric version. |
```
Comment (luhring) [inline docs/schema.md:650]: My two cents here — 

1. We don't have source packages at this time, so there's not a valuable semantic in distinguishing these here IMHO.
2. We don't have Debian-style version streams. We do have version streams that add suffixes to the package name, but it's not meaningful for the advisory data or for vulnerability matching at scan time — the package name should be used verbatim (even if a version stream suffix is present).

```suggestion
| `CGA` | The Chainguard package ecosystem; the `name` is the name of the package. |
```

Comment (cpanato) [reply docs/schema.md]: i've add that to the commit, i did similar text before, so i think we are aligned. i was just not sure

File: docs/schema.md (hunk 5 above)
Comment (oliverchang) [reply docs/schema.md]: Thanks. I think we still need to update the first column to be "Chainguard" instead? 

This table is referring to ecosystems, not ID prefixes.

Comment (cpanato) [reply docs/schema.md]: ah i see 🤦

Comment (cpanato) [reply docs/schema.md]: done

**Review by oliverchang — APPROVED** [review]

**Comment by cpanato** [comment]
cool, thanks for the review @oliverchang

**Comment by andrewpollock** [comment]
I note that OSV.dev is also importing [`Wolfi` ecosystem vulnerabilities](https://osv.dev/list?ecosystem=Wolfi) but it doesn't get a mention. I added validation for it in #282, but at the time didn't appreciate it wasn't in the documentation.

**Pull request merged**